oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/FormattingUtils.java

```diff
@@ -18,18 +18,28 @@
 
 import org.apache.jackrabbit.guava.common.base.Stopwatch;
 
-import java.time.LocalTime;
-import java.time.format.DateTimeFormatter;
 import java.util.concurrent.TimeUnit;
 
 public class FormattingUtils {
     public static String formatToSeconds(Stopwatch stopwatch) {
-        LocalTime seconds = LocalTime.ofSecondOfDay(stopwatch.elapsed(TimeUnit.SECONDS));
-        return DateTimeFormatter.ISO_TIME.format(seconds);
+        long seconds = stopwatch.elapsed(TimeUnit.SECONDS);
+        long absSeconds = Math.abs(seconds);
```

Comment:
That way we don't detect if the time went backwards, which may be the case in some edge case.

Reply:
Changed to print negative time spans: prefix with `-`.